home/static/css/welcome_page.css

```diff
@@ -20,12 +20,12 @@ body {
 
 a {
     background-color: transparent;
-    color: #71f4f5;
+    color: #5c0287;
```

Comment:
@sabderemane These color change aren't compatible with the current landing page. We need to box the landing page in its own stylesheet or find a color that works everywhere (probably unlikely considering the background on the landing page is black and white for the app)
![image](https://github.com/djangonaut-space/wagtail-indymeet/assets/1281215/328efdfb-63c9-4c06-bb0c-6c0b62677a43)